src/lib/stories.js

```diff
@@ -5,25 +5,34 @@ const client       = require('./client.js');
 const config       = require('../lib/configure.js').loadConfig();
 const log = console.log;
 
-const listStories = async (program) => {
-    debug('request workflows, members, projects, epics');
+async function fetchEntities() {
     let [projectsById, statesById, membersById, epicsById] = await Promise.all([
         client.listProjects().then(mapByItemId),
-        fetchStates().then(mapByItemId),
+        client.listWorkflows()
```

Comment:
Do you think it's worth adding labels to this entities fetch? That would remove the need to `await` them when calling `findLabels` later...